xtable-core/src/main/java/org/apache/xtable/delta/DeltaStatsExtractor.java

```diff
@@ -246,6 +246,7 @@ private static class DeltaStats {
     Map<String, Object> minValues;
     Map<String, Object> maxValues;
     Map<String, Object> nullCount;
+    boolean tightBounds;
```

Comment:
When parsing, you can add an option to the Jackson ObjectMapper to ignore unknown fields as well to avoid issues with future delta versions adding new fields our code is not aware of.

Reply:
Agree. I am assuming you are not suggesting removing `tightBounds` (and not taking any action for the comment) ?

Reply:
Can we add the option to the ObjectMapper as part of this PR to prevent new fields from causing failures in the future?

Reply:
I have added a catch all bucket for new stats that may exist in file stats. It should be worth reporting the new stats for debugging and for adding support in XTable sometime.